index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>AIME MALAIKA - Portofolio</title>
+  <link rel="stylesheet" href="assets/css/style.css">
+  <script src="https://use.fontawesome.com/75d401af0c.js"></script>
+</head>
+
+<body>
+  <header class="toolbar">
+    <div class="logo">
+      <a href="#">Welcome</a>
+    </div>
+    <div class="toggle-menu">
+      <input type="checkbox" id="nav-checkbox">
+      <label for="nav-checkbox" class="open-menu">
+        <i class="fa fa-bars"></i>
+      </label>
+      <nav class="navigation">
+        <label for="nav-checkbox" class="close-menu">
+          <i class="fa fa-close"></i>
+        </label>
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <section class="headline">
+    <div class="introduction">
+      <h1 class="greeting">
+        Hey there. I’m Aime
+      </h1>
+      <h1 class="describe">
+        I’m a software developer
+      </h1>
+      <p class="explanations">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+      <div class="social-icons">
+        <a href="#"><img src="./assets/images/twitter.svg" alt="tweeter icon"></a>
+        <a href="#"><img src="./assets/images/Linkedin.svg" alt="linkedin icon"></a>
+        <a href="#"><img src="./assets/images/medium.svg" alt="medium icon"></a>
+        <a href="#"><img src="./assets/images/github.svg" alt="github icon"></a>
+        <a href="#"><img src="./assets/images/angellist.svg" alt="angellist icon"></a>
+      </div>
```

Comment:
- [ ] You have done an excellent job with your project :muscle: but in this situation, I would suggest using the `<ul>` tag (unordered list) instead of the `<div>` and each `<a>` should be inside of the `<li>` tag (list item). Remember that the `<ul>` tag (unordered list) is the semantically correct way to "list" things like links, cards, etc :wink:

Reply:
thank for the comment i am going to work on this